modules/openapi/src/internals/GetExtensions.scala

```diff
@@ -49,16 +51,21 @@ object GetExtensions {
       }
       .toList
 
+  private val formatter =
+    DateTimeFormatter.ofPattern("yyyy-MM-dd").withZone(ZoneId.of("UTC"))
+
   protected[smithytranslate] def anyToNode(input: Any): Node = input match {
-    case null       => Node.nullNode()
-    case b: Boolean => Node.from(b)
-    case s: String  => Node.from(s)
-    case i: Int     => Node.from(i)
-    case d: Double  => Node.from(d)
-    case s: Short   => Node.from(s)
-    case l: Long    => Node.from(l)
-    case f: Float   => Node.from(f)
-    case n: Number  => Node.from(n)
+    case null                        => Node.nullNode()
+    case b: Boolean                  => Node.from(b)
+    case s: String                   => Node.from(s)
+    case i: Int                      => Node.from(i)
+    case d: Double                   => Node.from(d)
+    case s: Short                    => Node.from(s)
+    case l: Long                     => Node.from(l)
+    case f: Float                    => Node.from(f)
+    case n: Number                   => Node.from(n)
+    case d: java.time.OffsetDateTime => Node.from(d.toString())
+    case d: java.util.Date => Node.from(formatter.format(d.toInstant()))
```

Comment:
should we add java.time.Instant ?

Reply:
So far I only added the ones that I knew could be surfaced, afaik openapi only supports `date` and `date-time` formats out of the box, which are both covered here

Reply:
So far I only added the ones that I knew could be surfaced, afaik openapi only supports `date` and `date-time` formats out of the box, which are both covered here